jcon.00012/10.21105.jcon.00012.crossref.xml

```diff
@@ -0,0 +1,117 @@
+  <head>
+    <doi_batch_id>47734a9a394479d22533f985bb9a9c16</doi_batch_id>
+    <timestamp>20190712121036</timestamp>
+    <depositor>
+      <depositor_name>JOSS Admin</depositor_name>
+      <email_address>admin@theoj.org</email_address>
+    </depositor>
+    <registrant>The Open Journal</registrant>
+  </head>
+  <body>
+    <journal>
+      <journal_metadata>
+        <full_title>JuliaCon Proceedings</full_title>
+        <abbrev_title>JCON</abbrev_title>
+        <issn media_type="electronic">2642-4029</issn>
+        <doi_data>
+          <doi>10.21105/jcon</doi>
+          <resource>https://submissions.juliacon.org</resource>
+        </doi_data>
+      </journal_metadata>
+      <journal_issue>
+        <publication_date media_type="online">
+          <month>07</month>
+          <year>2019</year>
+        </publication_date>
+        <journal_volume>
+          <volume>1</volume>
+        </journal_volume>
+        <issue>1</issue>
+      </journal_issue>
+      <journal_article publication_type="full_text">
+        <titles>
+          <title>ChipSort: a SIMD and cache-aware sorting module</title>
+        </titles>
+        <contributors>
+          <person_name sequence="first" contributor_role="author">
+            <given_name>Nicolau</given_name>
+            <surname>Werneck</surname>
+            <ORCID>http://orcid.org/0000-0003-0017-5890</ORCID>
+          </person_name>
+        </contributors>
+        <publication_date>
+          <month>07</month>
+          <day>12</day>
+          <year>2019</year>
+        </publication_date>
+        <pages>
+          <first_page>12</first_page>
+        </pages>
+        <publisher_item>
+          <identifier id_type="doi">10.21105/jcon.00012</identifier>
+        </publisher_item>
+        <ai:program name="AccessIndicators">
+          <ai:license_ref applies_to="vor">http://creativecommons.org/licenses/by/4.0/</ai:license_ref>
+          <ai:license_ref applies_to="am">http://creativecommons.org/licenses/by/4.0/</ai:license_ref>
+          <ai:license_ref applies_to="tdm">http://creativecommons.org/licenses/by/4.0/</ai:license_ref>
+        </ai:program>
+        <rel:program>
+          <rel:related_item>
+            <rel:description>Software archive</rel:description>
+            <rel:inter_work_relation relationship-type="references" identifier-type="doi">https://doi.org/10.5281/zenodo.3254944</rel:inter_work_relation>
+          </rel:related_item>
+          <rel:related_item>
+            <rel:description>GitHub review issue</rel:description>
+            <rel:inter_work_relation relationship-type="hasReview" identifier-type="uri">https://github.com/JuliaCon/proceedings-review/issues/12</rel:inter_work_relation>
+          </rel:related_item>
+        </rel:program>
+        <doi_data>
+          <doi>10.21105/jcon.00012</doi>
+          <resource>https://submissions.juliacon.org/papers/10.21105/jcon.00012</resource>
```

Comment:
@vchuravy where do you want to the DOIs to resolve to?

Reply:
They should resolve to https://proceedings.juliacon.org/papers/

I am travelling this week so I don't know how long it will take for me to figure out how to deposit them there nicely.

Reply:
OK, well let's hold off the last steps here of registering the DOI until we have a mechanism for hosting these.

Reply:
What needs to be hosted there, for now I could just put a redirect there.

Can you point me towards the source-code that generates these PRs? We might be able to do something cute if we generate a database file as a json in the root directory that contains the information about all papers in the repository.